app/controllers/registration_controller.rb

```diff
@@ -12,17 +12,20 @@ def tracksheet
     render :plain => pdf.render, :content_type => :pdf
   end
 
-  def workflow_list
-    docs = Dor::SearchService.query(%(id:"#{params[:apo_id]}"))['response']['docs']
+  def workflows_for_apo(apo_id)
+    docs = Dor::SearchService.query(%(id:"#{apo_id}"))['response']['docs']
```

Comment:
me neither.  filed a new issue to investigate:  https://github.com/sul-dlss/argo/issues/950